go.mod

```diff
@@ -92,6 +92,7 @@ require (
 	github.com/prometheus/prometheus v1.8.2-0.20221021121301-51a44e6657c3 // @grafana/alerting-squad-backend
 	github.com/robfig/cron/v3 v3.0.1 // @grafana/backend-platform
 	github.com/russellhaering/goxmldsig v1.4.0 // @grafana/backend-platform
+	github.com/scottlepp/go-duck v0.0.15 // @grafana/backend-platform
```

Comment:
what team should this be?  I'm ok to change it to `@grafana/grafana-app-platform-squad` since we are generally starting to tackle the whole query service